controllers/testdata/supportbundle.json

```diff
@@ -0,0 +1,279 @@
+{
+  "apiVersion":"apps/v1",
+  "kind":"StatefulSet",
+  "metadata":{
+    "annotations": {
+      "deployment.kubernetes.io/revision": "1"
+    },
+    "name":"cdap-test-supportbundle",
+    "namespace":"default",
+    "creationTimestamp":null,
```

Comment:
FYI for expected output, please ensure only the expected output fields (and no extraneous) fields are included in the test template data, as there will be a fix to CDAP operator which changes the integration test JSON diff comparison behavior (see [CDAP-18732](https://cdap.atlassian.net/browse/CDAP-18732) for additional info).

Reply:
yeah I got the test result from the expected output